spec/client/latest

```diff
@@ -0,0 +1 @@
+v1.1.9
```

Comment:
These will be overridden by the SDK publish task, but need to be added manually here so the tests of this PR pass.